src/styles/main.scss

```diff
@@ -1,3 +1,11 @@
-body {
-  margin: 0;
-}
+@import "./utils/_extends";
+
+@import "./utils/_variables";
+
+@import "./blocks/header";
+
+@import "./blocks/page";
+
+@import "./blocks/stars";
+
+@import "./blocks/cards";
```

Comment:
Seems that you forgot to add these files to the commit. Use `git add .` for it
<img width="1253" alt="image" src="https://user-images.githubusercontent.com/99403198/216768370-e38f8b23-02ec-4b98-abfc-b775a7a21a13.png">